notebooks/official/migration/sdk-automl-video-classification-batch-prediction.ipynb

```diff
@@ -823,7 +792,7 @@
       "source": [
         "### Get test item(s)\n",
         "\n",
-        "Now do a batch prediction to your Vertex model. You will use arbitrary examples out of the dataset as a test items. Don't be concerned that the examples were likely used in training the model -- we just want to demonstrate how to make a prediction."
+        "Now do a batch prediction to your Vertex model. You use arbitrary examples out of the dataset as a test items. Don't be concerned that the examples were likely used in training the model -- you just want to demonstrate how to make a prediction."
```

Comment:
- change: "you just want to demonstrate how to make a prediction."
- to: "the purpose here is to demonstrate how to make a prediction."